agnostic-svelte-ts/.storybook/preview.js

```diff
@@ -0,0 +1,22 @@
+import '../src/app.css';
+import yourTheme from './yourTheme';
+import FontDecorator from '../src/lib/storybook-helpers/FontDecorator';
+
+export const parameters = {
+	docs: {
+		theme: yourTheme
+	},
+	actions: { argTypesRegex: '^on[A-Z].*' },
+	controls: {
+		matchers: {
+			color: /(background|color)$/i,
+			date: /Date$/
+		}
+	}
+};
+
+export const decorators = [() => FontDecorator];
+
+require('!style-loader!css-loader!../src/lib/css/common.resets.min.css');
+require('!style-loader!css-loader!../src/lib/css/common.properties.min.css');
+require('!style-loader!css-loader!../src/lib/css/common.utilities.min.css');
```

Comment:
Are these actually working properly in SB? I've historically found SB to be a real PITA for these globals and tracking latest. This is awesome if it's working ❤️

Reply:
Yes, they do! 🎉 I've tested adding a utility class (eg. `.h1` or `.m2`) to the Alert component and it seems that SB correctly bundles these styles! I've also saw that `app.css` was imported here in the svelte project, even though it didn't contain any additional rules. I've tested that that's still working in the TS project as well.